### DISCUSSION TITLE
Vapoursynth plugins: How to create a new formula for them that cannot meet the criteria by brew audit --new?

### DISCUSSION THREAD

**Discussion by Stefan-Olt** [Homebrew/discussions#3206] [Writing Formulae/Casks]
Hello,
there is Vapoursynth in brew, which has a lot of plugins (that have to be compiled) and it's also somewhat useless without plugins, because they bring the functionality, but only very few of them are in brew. I created formulae for some of them, but there are two issues:
- The GitHub repos are not notable enough
- Many are stable and mature and not actively maintained, with build issues on Apple Silicon (seems that most users are Windows and Linux), I can fork them and fix that, but forks are not accepted. I can include a patch in the formula, but that's also not accepted.

I don't think it's a good idea to have to compile all the plugins outside of brew, and link to the stuff compiled by brew. Let alone the work every person has to do again (if they are able to do that).

Any help what I should do?

Best regards
Stefan

**Comment by SMillerDev** [comment]
I think your best option is a personal tap. There you can compile whatever you want. The only difficulty (until github adds runners for it) would be the lack of apple silicon CI.

**Comment by Stefan-Olt** [comment]
Ok, but that seems to complicated and time-intensive for me. I hoped I could just contribute something to help others.
Seems the existing plug-ins got in at a time the requirements were a lot lower (the repo of vapoursynth-ocr has one fork and three stars)
